contrib/auto-completion/config.el

```diff
@@ -24,6 +24,10 @@ Possible values are `complete', `cycle' or `nil'.")
   "Provide a key sequence (string) to complete the current
 selection.")
 
+(defvar auto-completion-complete-with-key-timeout 0.1
```

Comment:
`delay` is better than `timeout`, consider this symbol name instead: `auto-completion-complete-with-key-sequence-delay`